permission/db.go

```diff
@@ -35,7 +35,16 @@ func (s *Store) getPermission(ctx context.Context, p1 Permission) (Permission, e
 }
 
 func (s *Store) addPermission(ctx context.Context, p1 Permission) (Permission, error) {
-	panic("addPermission not implemented")
+	var p Permission
+	stmt, err := s.db.PrepareNamedContext(ctx, "INSERT INTO people.permissions (name, description) VALUES (:name, :description) RETURNING permission_id, name, description")
```

Comment:
Any reason this isn't a `db.ExecContext`? dto for `addRole`

Reply:
`ExecContext` only returns the amount of rows affected and the id, this can return the entire object as requested in the `RETURNING` section

Reply:
In that case, any reason this isn't a `QueryContext`? ;)

Reply:
Exec is used for delete because I'm not wanting anything returned, other than an error if it fails

Reply:
But this isn't a DELETE, this is an INSERT, unless I'm missing something?

Reply:
Query doesn't have the `Named` part which allows for the `db` section of the struct

Reply:
I was giving context to why and where exec is used

Reply:
You can still use `db.NamedQueryContext`, no? It saves the second Execute call.

Reply:
Oh I see - there's no `NamedGetContext. That makes more sense now.

Reply:
It is slightly annoying and Query is slightly less nice to work with